fastai/vision/augment.py

```diff
@@ -259,14 +278,24 @@ def encodes(self, x:(Image.Image,TensorBBox,TensorPoint)):
 class RandomResizedCrop(RandTransform):
     "Picks a random scaled crop of an image and resize it to `size`"
     split_idx,order = None,1
-    def __init__(self, size, min_scale=0.08, ratio=(3/4, 4/3), resamples=(Image.BILINEAR, Image.NEAREST),
-                 val_xtra=0.14, max_scale=1., **kwargs):
+    def __init__(self,
+         size:(int, (int,int)), # Final size (after crop and resize), duplicated if one value is specified,,
+         min_scale:float=0.08, # Min scale of the crop
+         ratio:(float, float)=(3/4, 4/3), # Range of width/height of the output
```

Comment:
```suggestion
         ratio:(float, float)=(3/4, 4/3), # Range of (width, height) of the output
```

Reply:
I change it to # Range of width over height of the output. I think it's clearer this way right?